library/src/compute/calc.rs

```diff
@@ -962,6 +965,67 @@ pub fn quo(
     })
 }
 
+#[comemo::memoize]
+#[func]
+/// Returns a random number
+///
+///
+/// ## Example
+/// ```example
+/// #assert(calc.rand_int(seed: 1) == calc.rand_int(seed: 1))
+/// #assert(calc.rand_int(seed: 1) != calc.rand_int(seed: 2))
+/// ```
+///
+/// Display: Random integer
+/// Category: Calculate
+/// Returns: integer
+pub fn rand_int(
+    /// The seed, optional
+    #[named]
+    #[default(0x4d595df4d0f33173)]
```

Comment:
it is a bad idea to use this magic number 4 times in the code, maybe u should define a constant value like `default_seed`

Reply:
Is this for a change in behaviour, or just for readability?

Reply:
just for readability :)